internal/services/error_renderer.go

```diff
@@ -13,7 +13,7 @@ func writeBasicError(w http.ResponseWriter, r *http.Request, err error) {
 	if logger := ctxval.Logger(r.Context()); logger != nil {
 		logger.Error().Msgf("unable to render error %v", err)
 	}
-	w.Header().Set("Content-Type", "application/json")
+	w.Header().Set("Content-Type", "text/plain; charset=utf-8")
```

Comment:
shall we not render errors in json format?
I don't fully understand this change :thinking: is it relevant to the PR?

Reply:
Yeah I just noticed this and fixed it too.